folderTypes.cfg

```diff
@@ -4,13 +4,11 @@ Day:
     info:
         description: "text", 6
         animal_ID: "string"
-        
-        species: "list", ["Sim1-Cre_KJ18;AI14", "Tlx3-Cre_PL56;AI14", "Sst-IERS-Cre;Ai14", "Pvalb-IRES-Cre; Ai14", "Pvalb-IRES-Cre;Ai140; Rorb-2A-tTA2;Ai63", "Tlx3-Cre_PL56;Ai140; Pvalb-2A-FlpO;Ai65F", "Ctgf-2A-dgCre;Ai14", "Vip-IRES-Cre;Ai140;Pvalb-2A-FlpO;Ai65F", "Sst-IRES-Cre;Ai140;Pvalb-2A-FlpO;Ai65F", "Tlx3-Cre_PL56;Ai140;Sst-IRES-FlpO:Ai65F", "Sim1-Cre_KJ18;Ai140;Sst-IRES-FlpO;Ai65F", "Sim1-Cre_KJ18;Ai140;Pvalb-2A-FlpO;Ai65F", "Vip-IRES-Cre;Ai140;Sst-IRES-FlpO;Ai65", "Chat-IRES-Cre-neo;Snap25-LSL-F2A-GFP", "Slc17ae-IRES2-Cre;AI14"]
-
-        age: "string"
-        sex: "list", ['M', 'F']
-        weight: "string"
-        temperature: "list", ['34C', '32C', '25C', '37C', 'RT']
+        species: "list", ['mouse', 'human']
+        time_of_dissection: "string"
+        target_region: "list", ['', 'V1', 'ALM']
+        hemisphere: "list", ['', 'left', 'right']
```

Comment:
should this just pull from the specimen ID as well?

Reply:
Yes. I thought there might be situations where we can't recover the target hemisphere from LIMS, but I'mma go back on that.

Reply:
yeah the last numbers in the specimen ID (ex .06) should always indicate the hemisphere so I think we're good. And those also indicate the slice angle which we can use to retrieve the surface (medial/lateral) from a flipped or not flipped identifier